mlflow/genai/judges/prompts/correctness.py

```diff
@@ -30,6 +32,15 @@
 
 If the claim is fully supported by the document in the context of the question, you must say "The response is correct" in the rationale. If the claim is not fully supported by the document in the context of the question, you must say "The response is not correct"."""  # noqa: E501
 
+CORRECTNESS_TRACE_FALLBACK = f"""{CORRECTNESS_BASE_INSTRUCTIONS}
```

Comment:
@BenWilson2 Can we incorporate the correctness prompt suffix into the trace fallback as well?